dask/dataframe/io/parquet/arrow.py

```diff
@@ -27,14 +27,16 @@
 )
 
 # Check PyArrow version for feature support
-preserve_ind_supported = pa.__version__ >= LooseVersion("0.15.0")
+_pa_version = parse_version(pa.__version__)
+preserve_ind_supported = _pa_version >= parse_version("0.15.0")
 read_row_groups_supported = preserve_ind_supported
-if pa.__version__ >= LooseVersion("1.0.0"):
```

Comment:
These are not equivalent. Is there a reason to prefer to use `.major`?

Reply:
It's shorter, and one less parse operation to carry out.